src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -291,6 +291,42 @@ public void testGetIssues() throws Exception {
         String readRepoString = GitHub.getMappingObjectWriter().writeValueAsString(closedIssues.get(0));
     }
 
+    @Test
+    public void testQueryIssues() throws IOException {
+        List<GHIssue> openBugIssues = gitHub.getOrganization("hub4j-test-org")
+                .getRepository("testQueryIssues")
+                .queryIssues()
+                .state(GHIssueState.OPEN)
+                .label("bug")
+                .creator(gitHub.getMyself().getLogin())
+                .milestone("1")
+                .pageSize(10)
+                .list()
+                .toList();
+        assertThat(openBugIssues, is(not(empty())));
```

Comment:
Please make a few more meaningful assertions for each of the queries.  At least the number of issues and maybe a couple spot checks for issue values.